front/package.json

```diff
@@ -23,7 +23,7 @@
     "@amplitude/analytics-node": "^1.3.5",
     "@auth0/nextjs-auth0": "^3.5.0",
     "@dust-tt/client": "file:../sdks/js",
-    "@dust-tt/sparkle": "^0.2.265",
```

Comment:
make sure to merge and publish sparkle and switch back to main release before merging

Reply:
Yes definitely 👍🏼